kiwi/system/root_init.py

```diff
@@ -72,60 +73,42 @@ def create(self):
         """
         root = mkdtemp(prefix='kiwi_root.')
         try:
-            base_system_paths = [
+            base_system_paths = (
                 '/dev/pts',
                 '/proc',
                 '/etc/sysconfig',
                 '/var/cache',
                 '/run',
                 '/sys'
-            ]
+            )
             for path in base_system_paths:
                 Path.create(root + path)
                 Command.run(['chown', 'root:root', root + path])
 
-            Command.run(
-                ['mknod', '-m', '666', root + '/dev/null', 'c', '1', '3']
-            )
-            Command.run(
-                ['mknod', '-m', '666', root + '/dev/zero', 'c', '1', '5']
-            )
-            Command.run(
-                ['mknod', '-m', '622', root + '/dev/full', 'c', '1', '7']
-            )
-            Command.run(
-                ['mknod', '-m', '666', root + '/dev/random', 'c', '1', '8']
-            )
-            Command.run(
-                ['mknod', '-m', '644', root + '/dev/urandom', 'c', '1', '9']
-            )
-            Command.run(
-                ['mknod', '-m', '666', root + '/dev/tty', 'c', '5', '0']
-            )
-            Command.run(
-                ['mknod', '-m', '666', root + '/dev/ptmx', 'c', '5', '2']
-            )
-            Command.run(
-                ['mknod', '-m', '640', root + '/dev/loop0', 'b', '7', '0']
-            )
-            Command.run(
-                ['mknod', '-m', '640', root + '/dev/loop1', 'b', '7', '1']
-            )
-            Command.run(
-                ['mknod', '-m', '640', root + '/dev/loop2', 'b', '7', '2']
-            )
-            Command.run(
-                ['mknod', '-m', '666', root + '/dev/loop3', 'b', '7', '3']
-            )
-            Command.run(
-                ['mknod', '-m', '666', root + '/dev/loop4', 'b', '7', '4']
-            )
+            mknod_data = ("mknod -m 666 %s/dev/null c 1 3",
+                          "mknod -m 666 %s/dev/zero c 1 5",
+                          "mknod -m 622 %s/dev/full c 1 7",
+                          "mknod -m 666 %s/dev/random c 1 8",
+                          "mknod -m 644 %s/dev/urandom c 1 9",
+                          "mknod -m 666 %s/dev/tty c 5 0",
+                          "mknod -m 666 %s/dev/ptmx c 5 2",
+                          "mknod -m 640 %s/dev/loop0 b 7 0",
+                          "mknod -m 640 %s/dev/loop1 b 7 1",
+                          "mknod -m 640 %s/dev/loop2 b 7 2",
+                          "mknod -m 666 %s/dev/loop3 b 7 3",
+                          "mknod -m 666 %s/dev/loop4 b 7 4",
+                          )
+            for cmd in map(shlex.split, mknod_data):
```

Comment:
> If you could do that as part of this branch it would be really great, thanks for the effort

I'll do that, but I need some more time as I'm busy with other stuff.